src/ray/gcs/store_client/redis_store_client.cc

```diff
@@ -448,8 +448,16 @@ int RedisStoreClient::GetNextJobID() {
       "INCRBY", RedisKey{external_storage_namespace_, "JobCounter"}, {"1"}};
 
   auto *cxt = redis_client_->GetPrimaryContext();
```

Comment:
We can just make it `RedisStoreClient::AsyncGetNextJobID(Postable<void(int)> callback)`